core/controllers/suggestion.py

```diff
@@ -283,6 +283,22 @@ class UserSubmittedSuggestionsHandler(SuggestionsProviderHandler):
     suggestion type.
     """
 
+    URL_PATH_ARGS_SCHEMAS = {
+        'target_type': {
+            'schema': {
+                'type': 'basestring',
+            }
+        },
+        'suggestion_type': {
+            'schema': {
+                'type': 'basestring',
+            }
+        }
+    }
```

Comment:
There is probably a list of choices for both these types, please try to find them and use them here.